### PR TITLE
Documented the ExtensionPluginInterface

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

File: extending-contao4/managed-edition/extensions.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+is going to use. That's why you cannot simply have a `config.yml` with your own
+`security.firewalls` configuration, as the Symfony Config component will tell you
+
+> You are not allowed to define new elements for path "security.firewalls".
+
+Symfony makes sure, `security.firewalls` is specified only once. This is the only
+way it can ensure the order of firewalls is correct.
+
+Thanks to the `Contao\ManagerBundle\ContaoManager\Config\ExtensionPluginInterface` 
+you can, however, modify the all extension configurations of all the other bundles
```
Comment (discordier) [inline extending-contao4/managed-edition/extensions.md]:
```diff
-you can, however, modify the all extension configurations of all the other bundles
+you can, however, modify all extension configurations of all the other bundles
```

**Review by discordier — COMMENTED** [review]
LGTM aside from a small typo

File: extending-contao4/managed-edition/extensions.md (hunk 1 above)
Comment (discordier) [reply extending-contao4/managed-edition/extensions.md]: @leofeyer you missed this line?

Comment (leofeyer) [reply extending-contao4/managed-edition/extensions.md]: It does not seem wrong to me.

Comment (discordier) [reply extending-contao4/managed-edition/extensions.md]: o'rly? "modify the all extension configuration..."?
What is this ominous "all extension configuration" which I can modify and why does it belong to all the other bundles? :smile_cat:
Long story short, the first "the" is IMO superfluous.

**Comment by aschempp** [comment]
Merged and extended, please check the final result: https://docs.contao.org/books/extending-contao4/managed-edition/overriding-bundles.html

**Pull request closed without merging**